### ISSUE TITLE
IPP support

### ISSUE THREAD

**Issue by TheAssassin** [probonopd/WirelessPrinting#140]
Since this is for "printing", why not support IPP? It would be funny to add printers to your computer, and to send gcode to them this way. Even the signalling of the state of the printer (printing atm) could be supported, as well as a printer queue (although I'd make it semi automatic, i.e., require a button press or something before starting a new print, allowing the user to prepare the next print, then hitting a button to print the next job).

IPP uses HTTP/1.1 for transport, so it can be implemented on top of any web framework.

**Comment by probonopd** [comment]
Indeed. There is even a 3D printing extension for IPP if I am not mistaken.
